view/frontend/templates/js/replacejs.phtml

```diff
@@ -19,7 +19,12 @@ if (!$this->isActive()) return;
 
 <script type="text/javascript">
 
-!function () {
+// Store the configuration parameters passed from the php block
+// in the global object. Used in this file and on the payment page
+// in payment method renderer, vendor/boltpay/bolt-magento2/view/frontend/web/js/view/payment/method-renderer/boltpay.js
+window.boltConfig = <?=$this->getSettings(); ?>;
+
+window.onload = function () {
```

Comment:
`window.load` is a beast as it's a global shared attribute. Can we use `addEventListener` or `attachEvent` when possible?

ie., 
```
  if (window.addEventListener) {
    window.addEventListener("load", func, true);
  } else if (window.attachEvent) {
     window.attachEvent("onload", func);
   } else {
     window.onload = func;
   }
  }
```

(Actually I have a feeling the last else block is needed anymore as recent browsers support `addEventListener` but I'll leave it to you).

Reply:
Sure we can. I will change that.